net-analyzer/ossec-hids/files/makefile.patch

```diff
@@ -0,0 +1,12 @@
+--- a/Makefile	2018-10-20 17:24:07.270874551 +0200
++++ b/Makefile	2018-10-20 17:25:29.778878485 +0200
+@@ -378,7 +378,8 @@
+ install-server: install-server-generic
+ 
+ install-common: build
+-	./init/adduser.sh ${OSSEC_USER} ${OSSEC_USER_MAIL} ${OSSEC_USER_REM} ${OSSEC_GROUP} ${PREFIX}
++	# User creation is handled by Gentoo ebuild
```

Comment:
Please don't add commented out stuff in patches. Given the patch carries the original context, and is used only for the purpose of package builds, this only increases the patch size unnecessarily.